### ISSUE TITLE
add multi value gradient driver

### ISSUE THREAD

**Issue by epicEaston197** [Neos-Metaverse/NeosPublic#3657]
### Is your feature request related to a problem? Please describe.

this would be able to drive multiple values like float 3 and Float Q because I often use of value copy to connect 2 Value gradient drivers together 

### Relevant issues

#3654 but it's asking for a max value for this component

### Describe the solution you'd like

I want the value gradient driver to be able to drive multiple types of the same value and different values

### Describe alternatives you've considered

using multiple value gradient drivers and using value copies to connect them and if I want to use a value gradient driver float free with a float Q I have to Value copy to use both of them

### Additional context

for example I want to drive multiple float 3 values and float Q values being able to do this in one component like a multi-value gradient driver would be amazing I will be able to drive multiple float 3 values and float Q values that would clean this up amazingly and be so useful
![image](https://user-images.githubusercontent.com/76523692/164300050-79b8e0fb-cfa1-457d-aa27-769cf2c2d7b7.png)
![image](https://user-images.githubusercontent.com/76523692/164300082-a66fd558-6f8d-49da-9962-fe27401fb456.png)

**Comment by epicEaston197** [comment]
okay I wanted to add this note this component would basically be a container from multiple value gradient drivers

**Comment by Dirko3000** [comment]
And a ValueMultiDriver component driving all of the Progress values for all of your ValueGradientDrivers at the same time is not a good option because... ?

That's what I always do for this. A Panner1D (or whatever you want your progress driving input to be) driving a ValueMultiDriver that drives the progress bar for ALL OF my ValueGradientDrivers, no matter how many I have.
Very elegant solution already, and it sounds like you just want that ValueMultiDriver and a variable amount of ValueGradientDrivers to be merged into a single component for some reason?
I just don't really get it.

**Comment by epicEaston197** [comment]
> And a ValueMultiDriver component driving all of the Progress values for all of your ValueGradientDrivers at the same time is not a good option because... ?
> 
> That's what I always do for this. A Panner1D (or whatever you want your progress driving input to be) driving a ValueMultiDriver that drives the progress bar for ALL OF my ValueGradientDrivers, no matter how many I have. Very elegant solution already, and it sounds like you just want that ValueMultiDriver and a variable amount of ValueGradientDrivers to be merged into a single component for some reason? I just don't really get it.

yes I know this and I do this but it makes my stuff look really unorganized and messy
there has to be a way to compress this down so it looks nicer and more representable.

maybe one progress bar to drive multiple value gradient drivers?.
![image](https://user-images.githubusercontent.com/76523692/165114407-d8b6c6b3-3c1e-4cd8-9b00-77247b6206bc.png)

**Comment by Dirko3000** [comment]
If it's just a matter of being tidy, might a recommend either an inspector or utility object that allows you to minimize components in the inspector?

**Comment by epicEaston197** [comment]
> If it's just a matter of being tidy, might a recommend either an inspector or utility object that allows you to minimize components in the inspector?

I used the default inspector and it's way too inconvenient to spawn one out of my inventory because it will never be as convenient as clicking 2 on my keyboard

**Comment by Dirko3000** [comment]
I have an item that you place down, locally activate it, then all of the inspectors of any kind spawned after enabling it have collapsible components, even the default inspector. That sounds like it checks all the boxes in solving your issue. 
I can just give you the component collapsing cone.

**Comment by epicEaston197** [comment]
> 

yeah that sounds great is this in a public folder?

**Comment by Dirko3000** [comment]
> > 
> 
> yeah that sounds great is this in a public folder?

Uh... no, I just have it saved, I'll just give it to you next time I see you.
Oh right, worth mentioning that this is Sloppy McFloppy
